Utilities/Config.h

```diff
@@ -165,9 +165,16 @@ namespace cfg
 
 		bool from_string(std::string_view value, bool /*dynamic*/ = false) override
 		{
-			if (value == "false")
+			char copy[5];
+
+			if (value.size() <= 5)
+			{
+				std::transform(value.begin(), value.end(), std::begin(copy), ::tolower);
+			}
```

Comment:
```suggestion
			if (value.size() != 4 || value.size() != 5)
				return false

			char copy[5];
			std::transform(value.begin(), value.end(), std::begin(copy), ::tolower);
```

Reply:
This is a UB when the buffer is not sufficient.

Reply:
how is this undefined

Reply:
I didn't see the whole message.